<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParametersLowess</class>
 <widget class="QWidget" name="ParametersLowess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>5</y>
     <width>254</width>
     <height>120</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Smoothing Factor</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="paramSmoothingFac">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>This sets the proportion of data points that will 
be used to compute each regression estimate.
E.g. with a smoothing factor of 0.3, the regression 
at a given point x is computed using the 0.3*N 
nearest data points to x.</string>
      </property>
      <property name="decimals">
       <number>2</number>
      </property>
      <property name="minimum">
       <double>0.010000000000000</double>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>0.400000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Weighting Function</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="paramWeightingFunc">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>Weighting function used at each point a local 
fitting is performed. The tricube function is the 
standard weighting function used in the literature, 
but any function that has smooth contact with 
zero at normalizedDistance = 1 is fine.
The uniform weighting does not have this smooth 
contact and therefore produces regression 
estimates that lack local smoothness.</string>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>Tricube</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hann</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Uniform</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Fit Type</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="paramFitType">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>Try a linear fit first. If your data has a strong 
curvature and the linear fit provides a poor fit even 
at low smoothing factors, then use a quadratic fit.
An alternative is to transform the input data 
(e.g. 1/x) to straighten it before doing regression.</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <item>
       <property name="text">
        <string>Linear</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Quadratic</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Dimension Normalization</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="paramNormType">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>This only applies if your input data has more 
than one dimension.

When computing the distance to the nearest 
data points, it is important that the data in the 
different dimensions of the input be comparable.
This is the case e.g. if the dimensions are 
measurements of a position in space.

If the dimensions relate to different quantities, 
it is sensible to divide each dimension by a 
measure of its spread to make them comparable.
This is usually done by dividing by the standard 
deviation. The interquartile range (IQR) is a 
measure of spread that is more robust to outliers.</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Standard Deviation</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>IQR</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
